djangogirls/blog/views.py

```diff
@@ -1 +1,7 @@
 """Views for blog."""
+from django.shortcuts import render
+
+
+def post_list(request):
+    """Render post list on the webpage."""
+    return render(request, 'blog/post_list.html', {})
```

Comment:
Looks like you forgot to add the template for this to work

Reply:
Template in the next part of the tutorial, but

Reply:
The PRs, just like atomic commits, should be self-contained. Merging them should turn a project from a previously working state to a new working state. Adding a view that loads a non-exising template turns a project into an app with a broken URI handler which is not something that is tolerated in the real world. This is why you need to think about the features beyond how they are structured in the tutorial.

Reply:
At the time of commit creation, I didn't know how the features were structured beyond the tutorial. In the tutorials, first, add a URL, but I can commit it because I have pylint error and I need to add the view first. Further, I will combine the view, the template, and URL in one PR/commit, because these are interrelated changes. Also, It's strange when say do it according to the tutorial and then the tutorial is not correct, do otherwise. When I don't have "otherwise" instructions

Reply:
This is because the tutorial doesn't take into account the real-world development workflow as it has a different goal. I've missed that this PR doesn't contain the URL setup to connect this view (I should've waited). But you're right. Normally, you need to bundle all of the tightly connected parts into an atomic change that together make up a working feature. In the future, when you'll integrate testing, it'll become more obvious.